recipes/agent_windows.rb

```diff
@@ -7,4 +7,33 @@
 #
 # Recipe to install and configure the New Relic Infrastructure agent on Windows
 #
-raise 'Chef support for the Windows New Relic Infrastructure agent not yet available!'
+# Chef uses sha256 checksums, to generate this, I downloaded the file
+# and on OS X did 'shasum -a 256 newrelic-infra.msi' YMMV on other
+# platforms. It'd be *really* cool if NewRelic could provide an API
+# that you could ask for checksum values, or have checksums published
+# and available in some other way.
+#
+# TODO: make install location configurable
+
+windows_package 'newrelic-infra' do
+  action :install
+  source node['newrelic-infra']['windows_source'] if node['newrelic-infra']['windows_source']    
+  checksum node['newrelic-infra']['windows_checksum'] if node['newrelic-infra']['windows_checksum']
+  installer_type :msi
+end
+# Lay down newrelic-infra agent config
+template 'C:\Program Files\New Relic\newrelic-infra\newrelic-infra.yml' do
```

Comment:
This can be a `file` resource instead of a template. We should do it the same way we do in the Linux agent: https://github.com/newrelic/infrastructure-agent-chef/blob/master/recipes/agent_linux.rb#L69-L80